src/test/documentation_examples/mod.rs

```diff
@@ -1497,6 +1501,242 @@ async fn versioned_api_examples() -> std::result::Result<(), Box<dyn std::error:
     Ok(())
 }
 
+async fn aggregation_examples() -> GenericResult<()> {
+    let client = TestClient::new().await;
+    let db = client.database("aggregation_examples");
+    db.drop(None).await?;
+    aggregation_data::populate(&db).await?;
+
+    // Start Aggregation Example 1
+    let cursor = db
+        .collection::<Document>("sales")
+        .aggregate(
+            vec![
+                doc! { "$match" : { "items.fruit":"banana" } },
```

Comment:
I know this format matches the example provided in the DRIVERS ticket, but could we use `<key>: <value>` spacing for the key value pairs here and in other `doc!` invocations? That's what we use in our other existing existing examples.